release-notes/elastic-security/deprecations.md

```diff
@@ -12,30 +12,30 @@ Review the deprecated functionality for {{elastic-sec}}. While deprecations have
 
 % ::::{dropdown} Deprecation title
 % Description of the deprecation.
-% For more information, check [PR #](PR link).
+% For more information, refer to [PR #](PR link).
 % **Impact**<br> Impact of deprecation. 
 % **Action**<br> Steps for mitigating deprecation impact.
 % ::::
 
 ## 9.0.0 [elastic-security-900-deprecations]
-**Release date:** April 2, 2025
+**Release date:** April 8, 2025
 
 ::::{dropdown} Renames the `integration-assistant` plugin
 Renames the `integration-assistant` plugin to `automatic-import` to match the associated feature.
```

Comment:
Quick note that the backticks don't render correctly in the output. You may need to remove them and revise the plugin names to something more general.

<img width="593" alt="Screenshot 2025-04-04 at 2 46 05 PM" src="https://github.com/user-attachments/assets/7c52e20f-97ce-4bbe-9dc1-993b1793ad58" />

Reply:
I actually think they look okay. I know the formatting is not standard but at least it makes it clear they're snippets